### PR TITLE
adding json files

### PR DESCRIPTION
**Pull request by nvenkate-cyber**

### CODE REVIEW

File: portfolio/nandini.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "Hello! Nice to meet you, I'm Nandini Venkatesh!": {
```
Comment (Suavesito-Olimpiada) [inline portfolio/nandini.json]: I think you should have a like `"name"` as a key and then the name `Nanadini Venkatesh` as the value.

File: portfolio/nandini.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "What do I do?": "I major in Computer Science and Economics at the University of Maryland, College Park MD. My academic interests include cybersecurity, data analytics, and Machine Learning. I currently live in New Jersey!",
```
Comment (Suavesito-Olimpiada) [inline portfolio/nandini.json]: the same here, so the key is `"what_i_do"` as the key, in general I think the keys _should not_ be the same as the template, but they must be something simpler to write.

**Review by Suavesito-Olimpiada — COMMENTED** [review]
The structure that I think would be better so is flexible enough, but clear enough to put in the template is the next

```
{
    "name": "Nandini",
    "what_i_do": "description",
    "know_me": {
        "pasionate": ["etc...", "etc..."],
        "hobbies": {
            "main_hobbies": "I really like this...",
            "list_hobbies": ["fashion", "photography", ]
        }
        "work": {
            "skills": [
                {
                    "skill": "Java",
                    "description": "This is my most proficient language..."
                },
                {
                    "skill": "React",
                    "description": "Worked on creating Tempus"
                },
                {
                    "skill": "Arduino",
                    "description": ""
                }.
            ],
            "contact": {
                "email": "nandinivenkatesh485@gmail.com",
                "github": "nvenkate-cyber",
                "linkedIn": "nandini-venkatesh-979483175"
            }
        }
    }
}
```
Notice that in the GitHub and LinkedIn section we can insert the url in the template and with flask. :)

**Comment by Suavesito-Olimpiada** [comment]
The idea behind having two different ways to expres the hobbies is to allow having an introduction or a list, or both.

I'll change the template so it can hadle both cases if you think is not a lot of problem to change the formatting of the json. :V

**Comment by nvenkate-cyber** [comment]
I think having two different ways of expressing hobbies might just be repetitive. But I did add the other changes you suggested!

**Comment by Suavesito-Olimpiada** [comment]
The only questio that I have is, which format did you choose? list or decription? The profile template depends on it, hehe.

**Comment by nvenkate-cyber** [comment]
I chose description.

**Comment by Suavesito-Olimpiada** [comment]
Perfect, thanks! This looks awesome. :D

**Comment by nvenkate-cyber** [comment]
But the way I converted was I found a github project that converts markdown to json, and this was the way the markdown was to be converted.

**Comment by Suavesito-Olimpiada** [comment]
Ok, that makes a lot of sense. :+1:

**Comment by nvenkate-cyber** [comment]
@Suavesito-Olimpiada Can you also provide your hobbies so I can add that to the json files too?

**Comment by Reemeela2121** [comment]
This looks really good, Nandini! The JSON files look nicely formatted, too. Do you listen to BTS, by any chance? I've been a fan since 2015 LOL

**Review by Reemeela2121 — APPROVED** [review]
I think it looks good!

**Pull request merged**